packages/utilities/src/guards/Rate Limiter/RateLimit.ts

```diff
@@ -26,7 +26,7 @@ export function RateLimit(
   message = "message being rate limited!",
   rateValue = 1
 ): GuardFunction<BaseCommandInteraction | SimpleCommandMessage> {
-  function convertToMilli(timeValue: number, unit: TIME_UNIT): number {
+  function convertToMill(timeValue: number, unit: TIME_UNIT): number {
```

Comment:
I'd say `millis` is more correct, since `mill` is an existing, unrelated word

Reply:
> I'd say `millis` is more correct, since `mill` is an existing, unrelated word

that's my terrible spelling there! haha